cluster-operator/src/main/java/io/strimzi/operator/cluster/model/ZookeeperCluster.java

```diff
@@ -71,6 +78,8 @@
     public static final String ENV_VAR_ZOOKEEPER_CONFIGURATION = "ZOOKEEPER_CONFIGURATION";
     public static final String ENV_VAR_ZOOKEEPER_LOG_CONFIGURATION = "ZOOKEEPER_LOG_CONFIGURATION";
 
+    private Map<String, CertAndKey> certs;
```

Comment:
Javadoc describing the keys and values, please

Reply:
Do you mean just a comment, or really Javadoc ? Are we using it on private fields ? Btw I have to add the description to the `KafkaCluster` related fields as well, I guess.

Reply:
I think a javadoc comment makes it clearer that the comment is _intended_ as documentation (as opposed to commenting on some implementation detail, where I would be inclined to use `//`), so I personally would use a one line `/** Lorem ipsum dolor... */` comment. Obviously we don't (and shouldn't) have a policy on documenting *all* private fields, because for most of them it's self-evident what they are for. But I think where we (collectively) see value in documenting particular things then we should do so. 

Really I want to know what this field is used for, and right now it's name and type doesn't really help me enough in understanding that. All I know is that the key is a String -- but it is presumably a string with some particular purpose within the controller. So in cases like this it could be that a different name for the field is enough, no need for documentation, but if we can't think of a better name, then a sentence of doc should be sufficient.